proxy/src/telemetry/metrics/transport.rs

```diff
@@ -17,18 +17,18 @@ pub(super) type OpenScopes = Scopes<TransportLabels, Stamped<OpenMetrics>>;
 
 #[derive(Debug, Default)]
 pub(super) struct OpenMetrics {
-    open_total: Counter,
-    open_connections: Gauge,
-    write_bytes_total: Counter,
-    read_bytes_total: Counter,
+    pub open_total: Counter,
+    pub open_connections: Gauge,
+    pub write_bytes_total: Counter,
+    pub read_bytes_total: Counter,
```

Comment:
it's unfortunate that we have to change the interface for the tests.

Wdyt about having a 

```
#[cfg(test)]
impl OpenMetrics {
   ...
}
```

with helpers?

Reply:
Yeah, that sounds good. Will do.